scripts/servers-build.sh

```diff
@@ -17,11 +17,29 @@ build_servers_repositories() {
             cd go/src/github.com/mendersoftware/$repo
 
             case "$docker" in
-                generate-delta-worker|iot-manager|deployments|deployments-enterprise|deviceauth|deviceauth-enterprise|inventory|inventory-enterprise|tenantadm|useradm|useradm-enterprise|workflows|workflows-enterprise|create-artifact-worker|auditlogs|mtls-ambassador|deviceconnect|deviceconfig|devicemonitor|reporting)
+                iot-manager|deployments|deployments-enterprise|deviceauth|deviceauth-enterprise|inventory|inventory-enterprise|tenantadm|useradm|useradm-enterprise|workflows|workflows-enterprise|create-artifact-worker|auditlogs|mtls-ambassador|deviceconnect|deviceconfig|devicemonitor|reporting)
                     docker build -t $docker_url:pr .
                     $WORKSPACE/integration/extra/release_tool.py --set-version-of $docker --version pr
                     ;;
 
+                generate-delta-worker)
+                    local MENDER_BINARY_DELTA_VERSION=1.4.1
+                    local MENDER_ARTIFACT_VERSION=3.9.0
```

Comment:
These cannot be set here. We need to take them from the environment.

Reply:
No, I mean that you should your [this](https://github.com/mendersoftware/mender-qa/blob/master/.gitlab-ci.yml#L7) and [this](https://github.com/mendersoftware/mender-qa/blob/master/.gitlab-ci.yml#L43) (for this second one we need to translate to actual latest version. We can hard-code this one for now and make a note in [QA-446](https://tracker.mender.io/browse/QA-446))

Reply:
it is impossible out of the box `MENDER_BINARY_DELTA_VERSION` defaults to `"latest"` which is not a version, since you cannot download it.